packages/@okta/vuepress-site/guides/sign-into-spa/section10/react/getuserinfo.md

```diff
@@ -1 +1,9 @@
-placeholder
\ No newline at end of file
+User info ("claims") are returned from the `getUser()` method on the `Auth` object. The `Auth` object is made available in your components as `props.auth` via the `withAuth` Higher Order Component (HOC)
```

Comment:
needs a period :)

User info ("claims") are returned from the `getUser()` method on the `Auth` object. The `Auth` object is made available in your components as `props.auth` via the `withAuth` Higher Order Component (HOC).